src/diffusers/models/attention_processor.py

```diff
@@ -2229,44 +2229,45 @@ def __call__(
         for current_ip_hidden_states, scale, to_k_ip, to_v_ip, mask in zip(
             ip_hidden_states, self.scale, self.to_k_ip, self.to_v_ip, ip_adapter_masks
         ):
-            if mask is not None:
-                if not isinstance(scale, list):
-                    scale = [scale]
+            if scale > 0:
```

Comment:
`scale` can be a list when using image masking, no?

Reply:
Thanks for your careful inspect! This issues is resolved by adding an additional conditional statement.